lib/content-services/src/lib/content-node-share/content-node-share.dialog.ts

```diff
@@ -185,7 +188,7 @@ export class ShareDialogComponent implements OnInit, OnDestroy {
             });
     }
 
-    private createSharedLinks(nodeId: string) {
+    private createSharedLinks(nodeId: string | SharedLinkBodyCreate) {
```

Comment:
can you rename variable to make it matching to both types? Actually I think it does not match to SharedLinkBodyCreate type